src/gdef.cc

```diff
@@ -167,9 +165,6 @@ bool OpenTypeGDEF::ParseLigCaretListTable(const uint8_t *data, size_t length) {
       if (!subtable.ReadU16(&caret_format)) {
         return Error("Can't read caret values table %d in glyph %d", j, i);
       }
-      // TODO(bashi): We only support caret value format 1 and 2 for now
-      // because there are no fonts which contain caret value format 3
-      // as far as we investigated.
       if (caret_format == 0 || caret_format > kMaxCaretValueFormat) {
         return Error("bad caret value format: %u", caret_format);
       }
```

Comment:
You need to modify the `subtable.Skip(2)` below for format 3 as it has more data: https://docs.microsoft.com/en-us/typography/opentype/spec/gdef#caretvalue-format-3